src/react-apps/applications/receipt/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "receipt-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --config webpack.config.development.js --mode development --open",
+    "build": "cross-env NODE_ENV=production webpack --config webpack.config.production.js --progress",
+    "build-develop": "cross-env NODE_ENV=development webpack --config webpack.config.development --progress --watch",
+    "test": "jest",
+    "compile-ts": "tsc",
+    "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}",
+    "clean": "rimraf dist compiled",
+    "format": "prettier-tslint fix src/**/*.{ts,tsx}",
+    "webpack-watch": "cross-env NODE_ENV=development webpack --config webpack.config.development.js --watch --progress",
+    "lintfix": "tslint -c tslint.json 'src/**/*.{ts,tsx} --fix"
+  },
+  "author": "Altinn",
+  "license": "3-Clause BSD",
+  "dependencies": {
+    "@babel/core": "7.4.4",
+    "@material-ui/core": "4.3.2",
+    "moment": "^2.24.0",
+    "react": "16.8.6",
+    "react-content-loader": "4.2.1",
+    "react-dom": "16.8.6",
+    "react-grid-layout": "0.16.6",
+    "react-hot-loader": "4.3.1",
+    "react-modal": "3.4.5",
+    "react-redux": "7.1.0",
+    "react-router-dom": "4.3.1",
+    "redux": "4.0.0",
+    "redux-devtools-extension": "2.13.5",
+    "redux-logger": "3.0.6",
+    "redux-saga": "0.16.0",
+    "reselect": "3.0.1",
+    "webpack": "4.12.0",
+    "webpack-dev-server": "3.1.4"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.9.4",
+    "@types/jest": "23.3.14",
+    "@types/react": "16.8.10",
+    "@types/react-modal": "3.8.1",
+    "@types/react-redux": "7.0.1",
+    "@types/react-router-dom": "4.3.1",
+    "@types/react-dom": "16.8.3",
+    "@types/react-test-renderer": "16.8.1",
+    "@types/redux-mock-store": "^1.0.0",
+    "awesome-typescript-loader": "5.0.0",
+    "babel-jest": "^23.6.0",
```

Comment:
Should use same versions as in other app projects.

Reply:
Fixed